subprojects/packagefiles/libtiff/libtiff/meson.build

```diff
@@ -57,6 +57,13 @@ sources = files(
   'tif_@0@.c'.format(host_machine.system() == 'windows' ? 'win32' : 'unix'),
 )
 
+if host_machine.system() == 'windows' and meson.get_compiler('c').get_argument_syntax() == 'msvc'
```

Comment:
Not compatible with MSYS2?

Reply:
Personally I'd make the check less restrictive, but to be safe, I'm mirroring the one in the upstream CMakeLists.

Reply:
Fishy. I'd try just testing for windows.

Reply:
Okay, done.